mkosi/archive.py

```diff
@@ -63,6 +67,72 @@ def make_tar(src: Path, dst: Path, *, tools: Path = Path("/"), sandbox: Sequence
         )
 
 
+def make_oci(context: Context, dst: Path) -> None:
+    ca_store = dst / "blobs" / "sha256"
+    ca_store.mkdir(parents=True)
+
+    layer_archive = context.staging / context.config.output_tar
+    layer_digest = hash_file(layer_archive)
+    # Should we create a symlink at the old path such that the tar can still be used?
```

Comment:
I don't think we need to generate more output than people ask for, when they want tar, they can generate a tar.

Reply:
I was thinking of matching the current behavior for initrd/efi files which are also provided even when building disk images

Reply:
That's true, but the tar might be quite a bit bigger and we have an explicit tar output. In the end I'm only -0 about it, though.

Reply:
The tar is already created as it is used inside the OCI (also its currently not compressed, see comment further below in the code). So it would only be a symlink to an already existing file